keyboards/acheron/elongate/info.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
This info.json looks malformed, or is a VIA json instead.

Reply:
I think that in my twisted mind I made a VIA-QMK hybrid JSON. It was fixed now.